script/Arcueid/Day 7/07_00_ARC07_2_2.txt

```diff
@@ -34,6 +34,7 @@ Akiha's anger, having swollen like an inflating balloon,
 [sha:dfb27e28ee76a513a409b2bde68d4506820b564a]{
 -- Page 166, Offset 16982.
 -- 「……ちょっと頭いたい。自称吸血鬼だって言い張る女より、うちの妹の方が迫力あるとか……」
+//I don't know what to do about this line but I am suspect of 'Self proclaimed' and 'Doing my head in'
```

Comment:
This line is very weird and during the various TLCs we've discussed it quite thoroughly. As for the first part:
> ちょっと頭いたい
lit. "This kinda hurts my head."

and for the second:
> 自称吸血鬼だって言い張る女
lit. The woman who insists shes a self-proclaimed vampire

Which is kinda rough. This line is more-or-less accurate.